bitcoin/src/network/constants.rs

```diff
@@ -104,6 +104,24 @@ impl Network {
     pub fn magic(self) -> Magic {
         Magic::from(self)
     }
+
+    /// Converts a `Network` to its equivalent `bitcoind -chain` argument name.
+    ///
+    /// ```bash,no_run
```

Comment:
kek, if you hadn't specified `no_run` would rustdoc have tried to execute arbitrary bash code from a doccomment?

Reply:
I believe so lol, I don't think its a big security risk considering its ran locally but better safe than sorry. More so for copy/paste to verify where the options come from

Reply:
hah! That is my mistake. I'll fix in a follow up PR.

Reply:
I believe rustdoc doesn't understand anything else than `rust` so `bash` has no effect and without `no_run` it'd try to compile it as Rust code.

Reply:
I had a hunch that it just defaulted to `///rust` and if anything else was there it meant no default so not rust - hence it would not be run. And after trying it out, yes building the docs with no marker gives
```
    = note: error from rustc: unknown character escape: `-`
    = note: `#[warn(rustdoc::invalid_rust_codeblocks)]` on by default
help: mark blocks that do not contain Rust code as text
    |
107 |     /// ```text
    |            ++++
```
and running `cargo test --doc` fails. Appending any of `bash`, `no_run`, or `aoseunthao` clears the warning and stops the code being run and the tests all pass.

Reply:
When I added `bash` I had hoped that `bash` would give us syntax highlighting in the html docs but it does not. Bad Tobin for not checking originally.

Reply:
I'd still keep adding correct language names. You never know when they add the support. :)

Reply:
FYI: this was already requested: https://github.com/rust-lang/rust/issues/43165

Reply:
Yeah I suppose so, especially since we build docs with nightly.